src/main.js

```diff
@@ -3,22 +3,11 @@
  * SPDX-License-Identifier: AGPL-3.0-or-later
  */
 
-import Vue from 'vue'
+import { createApp, defineAsyncComponent } from 'vue'
 
 // Styles
 import './styles/styles.scss'
 import '@nextcloud/dialogs/style.css'
 
-Vue.prototype.t = t
-Vue.prototype.n = n
-Vue.prototype.OC = OC
-Vue.prototype.OCA = OCA
-
-export default new Vue({
-	el: '#notifications',
-	name: 'NotificationsApp',
-	components: {
-		NotificationsApp: () => import(/* webpackPreload: true */'./NotificationsApp.vue'),
-	},
-	render: (h) => h('NotificationsApp'),
-})
+const NotificationsApp = defineAsyncComponent(() => import(/* webpackPreload: true */'./NotificationsApp.vue'))
```

Comment:
Unrelated, but this comment doesn't work after Vite migration.
```suggestion
const NotificationsApp = defineAsyncComponent(() => import('./NotificationsApp.vue'))
```